contracts/os/andromeda-vfs/src/execute.rs

```diff
@@ -146,6 +146,7 @@ pub fn register_user(
     username: String,
     address: Option<Addr>,
 ) -> Result<Response, ContractError> {
+    ensure!(false, ContractError::TemporarilyDisabled {});
```

Comment:
```suggestion
#[cfg(not(test))]
ensure!(false, ContractError::TemporarilyDisabled {});
```

Does the above enable it for testing?

Reply:
Yeah this allows the unit tests to remain unchanged. Though the integration tests seem unaffected by it